setup.cfg

```diff
@@ -1,5 +1,45 @@
 [metadata]
-description-file = README.md
+name = libioc
+version = attr: libioc.__version__
+description = A Python library to manage jails with ioc{age,cell}.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url =  https://github.com/bsdci/libioc
+author = Stefan Grönke
+author_email = stefan@gronke.net
+python_requires = >=3.6
+platforms =
+    FreeBSD
+license = BSD 2-Clause License
+keywords =
+    FreeBSD
+    jail
+    ioc
```

Comment:
having itself as a keyword is pointless (on some platforms it's punished) but we can add iocage and iocell as keywords here